tests/acl/test_stress_acl.py

```diff
@@ -92,36 +166,69 @@ def setup_stress_acl_rules(rand_selected_dut, setup_stress_acl_table):
     rand_selected_dut.shell(cmd_rm_rules)
 
 
-def test_acl_add_del_stress(rand_selected_dut, setup_stress_acl_rules, get_function_conpleteness_level,
+def verify_acl_rules(rand_selected_dut, ptfadapter, ptf_src_port, ptf_dst_ports, dst_ip_addr, verity_status):
+    acl_nums = 0
+    while(acl_nums < ACL_RULE_NUMS):
+        acl_nums += 1
+        src_ip_addr = "20.0.0.{}".format(acl_nums)
+        pkt = testutils.simple_ip_packet(
+            eth_dst=rand_selected_dut.facts['router_mac'],
+            eth_src=ptfadapter.dataplane.get_mac(0, ptf_src_port),
+            ip_src=src_ip_addr,
+            ip_dst=dst_ip_addr,
+            ip_proto=47,
+            ip_tos=0x84,
+            ip_id=0,
+            ip_ihl=5,
+            ip_ttl=121
+        )
+
+        pkt_copy = pkt.copy()
+        exp_pkt = mask.Mask(pkt_copy)
+        exp_pkt.set_do_not_care_scapy(packet.Ether, 'dst')
+        exp_pkt.set_do_not_care_scapy(packet.Ether, 'src')
+        exp_pkt.set_do_not_care_scapy(packet.IP, "dst")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "ihl")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "tos")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "len")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "id")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "flags")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "frag")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "ttl")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "proto")
+        exp_pkt.set_do_not_care_scapy(packet.IP, "chksum")
+
+        ptfadapter.dataplane.flush()
+        testutils.send(test=ptfadapter, port_id=ptf_src_port, pkt=pkt)
+        if verity_status == "forward":
+            testutils.verify_packet_any_port(test=ptfadapter, pkt=exp_pkt, ports=ptf_dst_ports)
+        elif verity_status == "drop":
+            testutils.verify_no_packet_any(test=ptfadapter, pkt=exp_pkt, ports=ptf_dst_ports)
+
+
+def test_acl_add_del_stress(rand_selected_dut, tbinfo, ptfadapter, prepare_dst_ip, prepare_test_port,
+                            setup_stress_acl_rules, get_function_conpleteness_level,
                             toggle_all_simulator_ports_to_rand_selected_tor):   # noqa F811
+
+    ptf_src_port, ptf_dst_ports, dut_port = prepare_test_port
+
     normalized_level = get_function_conpleteness_level
+    cmd_add_rules = "sonic-cfggen -j {} -w".format(STRESS_ACL_RULE_JSON_FILE)
+    cmd_rm_rules = "acl-loader delete STRESS_ACL"
```

Comment:
The fixture `setup_stress_acl_rules` has added the rules, why do we need to add one more time?

Reply:
The fixture `setup_stress_acl_rules` seems unnecessary.

Reply:
Deleted